pkg/info/info.go

```diff
@@ -0,0 +1,55 @@
+package info
+
+import (
+	"encoding/json"
+	"fmt"
+	"os/exec"
+	"runtime"
+	"strings"
+	"unicode"
+
+	"github.com/weaveworks/eksctl/pkg/version"
+)
+
+//go:generate go run ./release_generate.go
+
+// Version holds versions info
+type Version struct {
+	EksctlVersion  string
+	KubectlVersion string
+	OS             string
+}
+
+// GetVersion returns versions info
+func GetVersion() Version {
+	return Version{
+		EksctlVersion:  version.String(),
```

Comment:
This returns quite a lot of stuff:
```
eksctl version: {"Version":"0.44.0","PreReleaseID":"dev","Metadata":{"BuildDate":"2021-03-26T15:34:25Z","GitCommit":"ee316d40"},"EKSServerSupportedVersions":["1.15","1.16","1.17","1.18","1.19"]}
```


I also think whatever is happening here is messing with the json (escaping everything):
```
$ ./eksctl info -o json
{"EksctlVersion":"{\"Version\":\"0.44.0\",\"PreReleaseID\":\"dev\",\"Metadata\":{\"BuildDate\":\"2021-03-26T15:34:25Z\",\"GitCommit\":\"ee316d40\"},\"EKSServerSupportedVersions\":[\"1.15\",\"1.16\",\"1.17\",\"1.18\",\"1.19\"]}","KubectlVersion":"v1.19.4","OS":"linux"}
```

I think some tests would help drive out desired UX in both these cases.

Reply:
@Callisto13 instead of returning all this info, would it be enough to just display the version?

Reply:
yes, ideally we would just show `eksctl version: 0.40.0` or similar

Reply:
@Callisto13 calling GetVersion() for retrieving eksctl's version returns a different version (version set in pkg/version/release.go). So I think we should go back to the previous implementation I had by executing the `eksctl version` command. Thoughts?
 ```
./eksctl info  
eksctl version: 0.44.0-dev
kubectl version: v1.19.7
OS: darwin

eksctl version
0.42.0
```

Reply:
I would rather avoid shelling out to ourselves if we don't have to. Can we add a func somewhere to get the exact thing we need internally?

Reply:
@Callisto13 🙋🏽‍♀️ Need some pointers on how I can extract the versions internally? Thanks!

Reply:
Hey! Just looking back and I realised I made a mistake 🤦‍♀️ : 

Above when you built and ran the command locally you got:

```sh
./eksctl info  
eksctl version: 0.44.0-dev
kubectl version: v1.19.7
OS: darwin
```

and thought the version was wrong as it used the version set in pkg/version/release.go. This was actually correct, since the version is a local dev build built from main, which will not be the same as if you just run `eksctl version`. using your installed release version.

So TLDR: you _can_ use `version.GetVersion()`, it will return the correct thing.